packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -32,6 +32,12 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   directionalHint?: DirectionalHint;
 
   /**
+   * How the element should be positioned in RTL layouts.
+   * If not specified, a mirror of `directionalHint` will be used instead
+   */
+  directionalHintForRtl?: DirectionalHint;
```

Comment:
is RTL TitleCased or CAPS?

i see `getRTL`, maybe we should be consistent.

https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

We need to get the TS coding guidelines moved into the fabric ghdocs folder.